app/javascript/components/new_project_button.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import RaisedButton from 'material-ui/RaisedButton';
+import ActionAndroid from 'material-ui/svg-icons/av/library-add';
+
+class New_Project_Button extends React.Component {

```

Comment:
I don't think we should mix underscores and CamelCase here. I'd prefer these to be named `NewProjectButton` instead...

Reply:
Perfect. My apologies. I'll change that right away.